llvm/lib/Target/RISCV/RISCVSubtarget.cpp

```diff
@@ -57,6 +57,10 @@ static cl::opt<unsigned> RISCVMaxBuildIntsCost(
 static cl::opt<bool> UseAA("riscv-use-aa", cl::init(true),
                            cl::desc("Enable the use of AA during codegen."));
 
+static cl::opt<unsigned> RISCVMinimumJumpTableEntries(
+    "riscv-min-jump-table-entries", cl::init(5), cl::Hidden,
```

Comment:
I don't think you need the default value based on how RISCVMinimumJumpTableEntries is used.